tddExercise/src/tddExercise/TaxValues.java

```diff
@@ -0,0 +1,16 @@
+package tddExercise;
+//common constants in our project
+public enum TaxValues {
```

Comment:
A simple constant helper class would have been sufficient instead of this enumeration. Only use what is needed now. Refactor to an enumeration only when and if you will make usage of its features.